src/main/kotlin/com/vk/kphpstorm/inspections/helpers/PhpDocPsiBuilder.kt

```diff
@@ -68,6 +70,13 @@ object PhpDocPsiBuilder {
         return field.docComment!!.addTag(project, "@var", "type")
     }
 
+    fun addTagToClass(klass: PhpClass, annotation: KphpDocTag): PhpDocTag {
+        val project = klass.project
+        val docComment = klass.docComment ?: createDocComment(project, klass)
+
+        return docComment.transformToMultiline(project).addTag(project, annotation.nameWithAt)
```

Comment:
Let's add tests with single-line phpdoc and multiline phpdoc with some other tag